docker/mongo/mongo-db-init.js

```diff
@@ -0,0 +1,6 @@
+db.createUser({
```

Comment:
We should move this to a `migrations` dir

Reply:
ye

Reply:
done

Reply:
Now t

> > Maybe move all of this inside a `server` dir so we can have the Unity project at the root level too
> 
> So we will go for a monorepo then? I was thinking of keeping the unity game in a separate project, but sure

Yup